### PR TITLE
chore: make logging consistent

### PR DESCRIPTION
**Pull request by wa0x6e**
- Remove redundant `console.log`
- Move the last few `log.error` to `log.warn`

### CODE REVIEW

File: src/helpers/shutter.ts [hunk 1]
```diff
@@ -107,7 +107,7 @@ export async function setProposalKey(params) {
     log.info(`[shutter] proposal scores updated for ${proposalId}`);
   } catch (e) {
     capture(e);
-    log.error(`[shutter] setProposalKey failed ${JSON.stringify(e)}`);
```
Comment (ChaituVR) [inline src/helpers/shutter.ts:110]: Maybe we can leave it an error. Slightly more important

**Review by ChaituVR — APPROVED** [review]
utAck

File: src/helpers/shutter.ts (hunk 1 above)
Comment (wa0x6e) [reply src/helpers/shutter.ts:110]: In that case, we should also move all `log.warn` to `log.error`, unless they're validation error (in ingestor).

Comment (wa0x6e) [reply src/helpers/shutter.ts:110]: Let's not forget we also capture the error on Sentry, to that log should be gone

**Pull request merged**